### DISCUSSION TITLE
Approved for GitHub Global Campus but Haven't Received Benefits Yet

### DISCUSSION THREAD

**Discussion by vladigpasev** [community/community#132236] [GitHub Education] [answered]
### Hi everyone,

I was approved for the GitHub Global Campus three days ago (last Friday), but I still haven't received my benefits. Has anyone else experienced this delay? Is there a usual waiting period, or should I be taking any further steps to ensure I receive the benefits?

I've already checked the common FAQs and previous threads but couldn't find a solution to my specific issue. Any advice or guidance would be greatly appreciated!

Thanks in advance for your help!

**Comment by davevad93** [comment, marked as answer]
Hi @vladigpasev ,

If the status label on your application is purple and states "`Congrats! Your request for @vladigpasev was approved and your benefits are now available.`", technically it means that you can access your benefits, so first try to go to the [Education learn page](https://education.github.com/learner/learn) , on the right side check if you have claimed offers:

![1](https://github.com/user-attachments/assets/510c7e2f-9357-4910-bffe-1af610226644)

Alternatively you can go [here](https://education.github.com/pack/redeem/copilot-student) to claim your free `GitHub Copilot` subscription that comes with the `Student Developer Pack`.

If you are unable to access `Copilot` there, try with installing the `Copilot` extension in your IDE e.g. `VS Code`, `Visual Studio`, `IntelliJ`, `Android Studio`, `R Studio` etcetera.

The following steps are for `VS Code`:

1. Install the Copilot extension in `VS Code`.
2. Log in with your current GitHub account.
3. Copy the 8-digit verification code to the clipboard and paste it into the Device Activation pop-up window.

If you already have the extension installed in your IDE, try logging out and logging back in.

**Reply by vladigpasev** [reply]
Yes, now I can access them, thank you very much. I've got one more question, regarding Copilot, I already have a Copilot subscription, which I pay (10 usd per month). How do I switch to the education version which is free? @davevad93

**Reply by davevad93** [reply]
You're welcome @vladigpasev , regarding switching `Copilot` from paid to free I don't know how to do it, I guess if you terminate your paid `Copilot` subscription it will automatically switch to the one that comes with the `Student Developer Pack`, but I'm not sure, so I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or receipts` and ask about it.

**Comment by ketikagarg** [comment]
Same here! I was verified last Friday and I have not got any benefits yet. I can't re-access Github Copilot either.
